pkgs/development/python-modules/python-vipaccess/default.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv
+, buildPythonPackage
+, fetchPypi
+, lxml
+, oath
+, pycryptodome
+, requests
+}:
+
+buildPythonPackage rec {
+  pname = "python-vipaccess";
+  version = "0.10.3";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1m6b7qipiaj6pz86kjhyq5m5jxxijpk58gpsdkj5bn0wjl6x1pg2";
+  };
+
+  propagatedBuildInputs = [
+    lxml
+    oath
+    pycryptodome
+    requests
+  ];
+
+  # requires a network connection
+  doCheck = false;
```

Comment:
```
  checkInputs = [ pytest ];
  checkPhase = ''
    mv vipaccess vipaccess.hidden
    pytest tests/ -k 'not (test_check_token_detects_valid_totp_token
      or test_check_token_detects_valid_hotp_token
      or test_check_token_detects_invalid_token)'
  '';
```

Reply:
Ah, thanks, I didn't know how to do that.  I shortened the `-k` argument to `-k not test_check_token`, though. Works on NixOS.

Reply:
Yes I forget it's a pattern match.